src/arrays.js

```diff
@@ -1,61 +1,73 @@
 const getNthElement = (index, array) => {
-  // your code here
+  
+  if (index >= array.length){
+    return array[index-array.length];
+  }
+  return array[index];
 };
 
 const arrayToCSVString = array => {
-  // your code here
+const csv = array.toString();
+return(csv);
 };
 
 const csvStringToArray = string => {
-  // your code here
+const array = string.split(',');
+return(array);
 };
 
 const addToArray = (element, array) => {
-  // your code here
+array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+return array.concat(element);
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+array.splice(index, 1);
 };
 
 const numbersToStrings = numbers => {
-  // your code here
+return numbers.map(String);
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
+
 };
 
 const reverseWordsInArray = strings => {
-  // your code here
+
 };
```

Comment:
As above, as there is no code here I'll just provide a solution to this problem. Have a play around with it and see if you can work out how/why it works.
```
const reverseWordsInArray = strings => {
  return strings.map(string => {
    return string
      .split('')
      .reverse()
      .join('');
  });
};
```